### ISSUE TITLE
Icalendar::Event.parse'ing incorrectly

### ISSUE THREAD

**Issue by jmera** [icalendar/icalendar#152]
Steps to duplicate:

```ruby
ics_str = URI.parse("https://db.tt/2cZqLLov").read
ical_events = Icalendar::Event.parse(ics_str)
ical_events.map(&:uid) # => ["uid-1234-uid-4321", "uid-1234-uid-4321"]
```

The same UID is returned for each event. If you inspect [the .ics](https://db.tt/2cZqLLov) in your text editor, it's fairly easy to see the uid output does not match what's in the file.

Am I missing something here?

**Comment by tfl** [comment]
Same problem here. Its not only the UID but the same event gets repeated for the number of events in the file. I have 12 events in my calendar - thus 12 times the same summary, description, etc. pp. See #153

**Comment by jmera** [comment]
@tfl good catch, I didn't go as far as checking every single event attribute.

**Comment by tfl** [comment]
But did you read what I added to #153 ? I see it as my fault and/or as a bug in Radicale (from which I migrated).

**Comment by jmera** [comment]
@tfl yes I read your issue. I still think this a bug—you can clearly see from the .ics I link to that each event has a unique UID yet is being parsed incorrectly.  As far as I know, that .ics is formatted correctly, unlike the one you describe in #153.

**Comment by tfl** [comment]
You are right. I tested it against my (new imported) data in Davical (which works greate with DavDroid) and I can confirm that parsing the calendar produces the exact same UID for every entry. In fact, every parsed event has now the UID from the last event in the string that was returnd by the server. Not the first one, for which I would have some sort of understanding ;)

**Comment by schnika** [comment]
I looked into this issue and i think I found a solution to this problem. see #156

**Comment by rahearn** [comment]
The `parse` method on `Event` can only handle a single `Event` object currently. If you want to Parse multiple events, I recommend wrapping them in a `VCALENDAR` object and parsing with `ICalendar::Calendar.parse`

**Comment by rahearn** [comment]
Fixed by #156
